src/Brzozowski/Derivatives of Regular Expressions - Janusz A Brzozowski.md

```diff
@@ -24,15 +24,15 @@ We are concerned with:
 
 # 2. Regular Expressions
 
-A regular expression represents a set of sequences. We define the following operations on sets of sequences: If $P$ and $Q$ are two sets of sequences of symbols from our alphabet, $\Sigma_k$, we have:
+A regular expression defines a language. We define the following operations on languages: If $P$ and $Q$ are two languages of symbols from our alphabet, $\Sigma_k$, we have:
```

Comment:
I think we can still say "represents a `set of strings`", in the places where the paper says `set of sequences`, as it is important to still think of it as a `set`.  And we can keep the place in the paper you mentioned where `set of sequences` is referred to as a language or even add some, that say `A regular expression represents a set of strings or language for short.`  What do you think?

Reply:
Maybe we just make a separate definition of _language_ as a set of finite strings and then define everything in terms of that? 

So here we'd say "represents a language".

I'm hoping we can get rid of the _set_ word as part of the definition at some point. I don't think we ever use the fact that this is a set anywhere and as good intuitionalists we should try to avoid it.

I'd be in favour of keeping "set of sequences" if your intention is to keep this document as close as possible to the paper (and the paper uses this term)?

Reply:
Okay I now see where you are coming from, that is an interesting angle and I like it.  Lets embrace the intuitionalist angle and make this paper fit it :)

Reply:
But then we need to address the `emptyset` as well, would that then become an empty language?